File: backend/app/schemas/alert.py
from pydantic import BaseModel
from typing import Optional

class Alert(BaseModel):
    id: str
    facility_id: str
    facility_name: str
    severity: str  # CRITICAL, HIGH, MODERATE, SAFE
    title: str
    message: str
    timestamp: str
    acknowledged: bool = False
    resolved: bool = False
    recommended_action: Optional[str] = None

File: frontend/src/types/alert.ts
export interface Alert {
  id: string;
  facility_id: string;
  facility_name: string;
  severity: 'SAFE' | 'MODERATE' | 'HIGH' | 'CRITICAL';
  title: string;
  message: string;
  timestamp: string;
  acknowledged: boolean;
  resolved: boolean;
  recommended_action?: string;
}
